cogs/games.py

```diff
@@ -259,6 +320,22 @@ def _calculate_score(self, pot, time_spent):
 			ctx.guild.id, ctx.author.id, answered_at, question_hash, result
 		)
 
+	@trivia.command()
+	async def categories(self, ctx):
+		'''Get a list of valid categories for the trivia command.'''
+		# Just dump the list of valid categories into an embed and send it
+		description = ''
+
+		for category_name in TRIVIA_CATEGORIES:
```

Comment:
Good point, I've moved the code for building the category list into `get_trivia_categories`, since the list only needs to be built once, and made a default for it that tells the user there was an error getting the category list.